testsuite/test_meeting_processing_file_list.py

```diff
@@ -0,0 +1,127 @@
+from flask import url_for
+from pyquery import PyQuery
+
+from mrt.models import db
+
+from .factories import RoleUserMeetingFactory
+from .factories import RoleFactory, JobFactory, MeetingFactory
+
+
+def test_meeting_manager(app):
+    meeting = MeetingFactory()
+
+    first_role = RoleFactory(permissions=('manage_meeting',))
+    second_role = RoleFactory(permissions=('view_participant',))
+
+    first_role_user = RoleUserMeetingFactory(role=first_role,
+                                             meeting=meeting)
+    second_role_user = RoleUserMeetingFactory(role=second_role,
+                                              meeting=meeting)
+
+    first_job = JobFactory(user_id=first_role_user.user.id,
+                           meeting_id=first_role_user.meeting.id)
+    second_job = JobFactory(user_id=second_role_user.user.id,
+                            meeting_id=first_role_user.meeting.id)
+
+    db.session.add(first_job)
+    db.session.add(second_job)
+    db.session.commit()
+
+    client = app.test_client()
+    with app.test_request_context():
+        with client.session_transaction() as sess:
+            sess['user_id'] = first_role_user.user.id
+
+        resp = client.get(url_for('meetings.processing_file_list',
+                                  meeting_id=meeting.id))
+        assert resp.status_code == 200
+        html = PyQuery(resp.data)
+        assert html('#job-list tbody tr').length == 2
+
+
+def test_superuser(app):
+    meeting = MeetingFactory()
+
+    first_role = RoleFactory(permissions=('',))
```

Comment:
A user without permissions simply does not have a Role assigned. A role containing 0 permissions doesn't make much sense.